lectures/_config.yml

```diff
@@ -68,6 +68,8 @@ sphinx:
       description: This website presents a set of lectures on advanced quantitative economic modeling, designed and written by Thomas J. Sargent and John Stachurski.
       keywords: Python, QuantEcon, Quantitative Economics, Economics, Sloan, Alfred P. Sloan Foundation, Tom J. Sargent, John Stachurski
       google_analytics_id: UA-54984338-11
+      launch_buttons:
```

Comment:
@AakashGfude we have `mybinder` support on the advanced lecture site. I am surprised we need a `launch_buttons` key in the config. Can you check that?

Reply:
@mmcky we have binder support by default, even if we don't specify any `launch_buttons` config. https://github.com/QuantEcon/quantecon-book-theme/blob/master/quantecon_book_theme/launch.py#L86 . For others we will have to specify.

Reply:
thanks @AakashGfude we should probably change that to an enabled option.